core/cppgc.rs

```diff
@@ -1,74 +1,89 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
+use crate::JsRuntime;
 use std::any::TypeId;
+pub use v8::cppgc::GarbageCollected;
```

Comment:
nitpick: should've been `GcResource` or `GarbageCollectable`

Reply:
I agree we should keep it as `GcResource`